frontend/src/context/anticipatoryAction/AAStormStateSlice/types.ts

```diff
@@ -4,60 +4,64 @@ import { DateItem } from 'config/types';
 export enum AACategory {
```

Comment:
Let's make this StateSlice more different from the AADrought

Reply:
I'm not sure what you mean, is it the naming with "category" that can be confusing because it's also used in drought ?
maybe i can change by "expositionLevel" or someting like that

Reply:
Yes in general I think we need to make the two slices a bit more different. At the moment it's a bit confusing and we even have functions with the same name but for different AA type. It's a bit my fault to be honest because I advised you to copy the code from AADrought when actually I think AAStorm is more similar to a standard layer than to AADrought. Anyhow, we can address that in a follow up PR

Reply:
Yes, I understand. I believe it will soon become clearer how to create the appropriate state functions for a better structure tailored to AAStorm. At the time, it wasn't entirely clear to me how to completely restructure this, but we can revisit and refine this part to make it more suitable. 
If that works for you, I suggest merging this PR without this change into a reference branch so everyone can sync with the latest version of our developments. We can make this change in a follow up PR like you suggested